packages/extension/src/ui/features/ledger/utils.ts

```diff
@@ -113,3 +117,9 @@ export class LedgerAlephium extends AccountDiscovery {
     return newDiscoveredAccounts
   }
 }
+
+const checkKeyType = (keyType: string) => {
+  if (keyType !== "default") {
```

Comment:
It seems we can support the `gl-secp256k1` key type in the ledger wallet. I'll test it and create a new PR.

Reply:
Nice, thanks :+1: